Externals/Bochs_disasm/CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
-set(SRCS dis_decode.cc
-         dis_groups.cc
-         resolve.cc
-         syntax.cc)
+add_library(bdisasm STATIC
+  dis_decode.cc
+  dis_groups.cc
+  resolve.cc
+  syntax.cc
+  $<$<BOOL:WIN32>:stdafx.cc>
```

Comment:
I think the `if() target_sources(bdisasm) endif()` syntax should be preferred for readability.